adventure/api.py

```diff
@@ -64,4 +65,14 @@ def move(request):
 @api_view(["POST"])
 def say(request):
     # IMPLEMENT
-    return JsonResponse({'error':"Not yet implemented"}, safe=True, status=500)
+    data = json.loads(request.body)
+    rsp = data['message']
+    user = request.user
+    player = user.player
+    room = player.room()
+    player_id = player.id
+    currentPlayerUUIDs = room.playerUUIDs(player_id)
+    for p_uuid in currentPlayerUUIDs:
+        pusher.trigger(f'p-channel-{p_uuid}', u'broadcast', {'say':f'{player.user.username} says {rsp}'})
+    return JsonResponse({'say':f'{player.user.username} says {rsp} ids are {currentPlayerUUIDs}'}, safe=True)
+    
```

Comment:
This is good, it's only a small step to implement the shout command beyond this one, I won't give it away though.